parser.y

```diff
@@ -1568,6 +1586,9 @@ PlacementOption:
 	{
 		$$ = &ast.PlacementOption{Tp: ast.PlacementOptionLearnerConstraints, StrValue: $3}
 	}
+
+PlacementOption:
+	DirectPlacementOption
 |	"PLACEMENT" "POLICY" EqOpt stringLit
```

Comment:
Is `PlacementOption` used now? May be below is better because a database or table can not assign a policy and set the direct placement option at the same time.

```
PlacementOptionListOrPolicy:
	PlacementOptionList
|       "PLACEMENT" "POLICY" EqOpt stringLit
```

Reply:
`DirectPlacementOption` is a basic option set for create policy usage.
`PlacementOption` is super set of `DirectPlacementOption`, plus placement policy for create table usage.

`a database or table can't assign a policy and set the direct placement option at the same time`, @morgo Is this true

Reply:
Yes that's true. For simplification we decided setting a placement policy unsets all placement options (and vice versa)

Reply:
After code trying. we couldn't convert `PlacementPolicyOrOptionList` to a single database option since databaseOptList assume it's element is every single database option, no a list here.

for example
`placement policy = x character set='utf8' follower_constraints='yy'`
parser will treat every token individually, every element is a database option, we couldn't divide/tell the placement_options list and placement_policy apart at the first beginning. For this case character_set is not a placement option.

Reply:
I think you are right, we should at least guarantee `follower=1 character set='utf8' follower_constraints='yy'` can be accepted by parser.